meinberlin/react/contrib/PaginationButton.jsx

```diff
@@ -6,14 +6,19 @@ export const PaginationButton = (props) => {
     : undefined
 
   const itemClass = props.isActive
-    ? 'pagination__item active'
-    : 'pagination__item'
+    ? 'active'
```

Comment:
the tests fail because you changed this to just `active` so you have to change it in the test as well